app/components/schema-field-radio/component.js

```diff
@@ -0,0 +1,38 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  init() {
+    this._super(...arguments);
+    let key = this.get('key');
+    let document = this.get('document');
+    let defaultValue = this.get('property.default');
+    let initialValue =  '';
+
+    if(typeof defaultValue !== 'undefined') {
+      initialValue = defaultValue;
+    }
+
+    if(typeof document.get(key) !== 'undefined') {
```

Comment:
Can we tweak this to only call `document.get` once (it currently calls `document.get` once for the guard and again below to grab the value)?